### PR TITLE
dsg-1422: add wm publisher for init docs processing

### PR DESCRIPTION
**Pull request by jgnuvalence**
## Description

[___JIRA_TICKET_TITLE___](//jira_url)

## Types of changes

Patch

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
  Minor
- [ ] New feature (non-breaking change which adds functionality)
  Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [ ] I have confirmed all acceptance criteria defined by the story have been met.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ allprojects {
 	version = '0.0.1-SNAPSHOT'
 
 	repositories {
+		mavenLocal()
```
Comment (Ortega-Dan) [inline build.gradle:15]: I believe this is better to remove after developing ?

Comment (jgnuvalence) [reply build.gradle:15]: ups, thank you, I'll remove it

Comment (jgnuvalence) [reply build.gradle]: removed

File: helm/document-management/templates/deployment.yaml [hunk 2]
```diff
@@ -178,8 +178,7 @@ spec:
             name: clamav-nginx
 {{- end }}
 ---
-{{- if .Values.env.PUBSUB_ENABLED }}
-{{- if .Values.env.PUBSUB_EMULATOR_HOST }}
+{{- if .Values.env.PUBSUB_EMULATOR_ENABLED }}
```
Comment (Ortega-Dan) [inline helm/document-management/templates/deployment.yaml:181]: Nice you are fixing this, made me curious and realized this var changed some commits ago, but I realize we need to adjust it in this 2 other places:
(one is functional the other is just documentation)
1) https://github.com/Nuvalence/dsgov-document-management/blob/b9b102f34b23eb1d57e08d73c013a4a439e86155/README.md#L292-L292
2) https://github.com/Nuvalence/dsgov-document-management/blob/b9b102f34b23eb1d57e08d73c013a4a439e86155/service/src/main/resources/application.yaml#L62-L62

Comment (jgnuvalence) [reply helm/document-management/templates/deployment.yaml:181]: I made the requested changes

**Review by Ortega-Dan — APPROVED** [review]

**Pull request merged**